main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -189,29 +189,29 @@ static class EqualShapeDistanceSameCoordinatesNotice extends ValidationNotice {
   @GtfsValidationNotice(severity = ERROR, files = @FileRefs(GtfsShapeSchema.class))
   static class EqualShapeDistanceDiffCoordinatesNotice extends ValidationNotice {
 
-    // The id of the faulty shape.
+    /** The id of the faulty shape. */
     private final String shapeId;
 
-    // The row number from `shapes.txt`.
+    /** The row number from `shapes.txt`. */
     private final int csvRowNumber;
 
-    // The faulty record's `shape_dist_traveled` value.
+    /** The faulty record's `shape_dist_traveled` value. */
     private final double shapeDistTraveled;
 
-    // The faulty record's `shapes.shape_pt_sequence`.
+    /** The faulty record's `shapes.shape_pt_sequence`. */
     private final int shapePtSequence;
 
-    // The row number from `shapes.txt` of the previous shape point.
+    /** The row number from `shapes.txt` of the previous shape point. */
     private final long prevCsvRowNumber;
 
-    // The previous shape point's `shape_dist_traveled` value.
+    /** The previous shape point's `shape_dist_traveled` value. */
     private final double prevShapeDistTraveled;
 
-    // The previous record's `shapes.shape_pt_sequence`.
+    /** The previous record's `shapes.shape_pt_sequence`. */
     private final int prevShapePtSequence;
 
     // Actual distance traveled along the shape from the first shape point to the previous shape
-    // point.
+    /** point. */
```

Comment:
```suggestion
    /** Actual distance traveled along the shape from the first shape point to the previous shape point. */
```
** Maybe it is too long for the formatting tool.